src/SprykerEco/Yves/CrefoPay/Theme/default/components/molecules/crefopay-form-loader/crefopay-form-loader.twig

```diff
@@ -0,0 +1,26 @@
+{% extends model('component') %}
+
+{% define config = {
+    name: 'crefopay-form-loader',
+    tag: 'crefopay-form-loader'
+} %}
+
+{% define data = {
+    shopPublicKey: '',
```

Comment:
Why do you need that data? If you can only leave only attribute.

Reply:
The only reason I see - to make it required. And as I see it should be required.